frappe/public/js/frappe/form/controls/text_editor.js

```diff
@@ -1,5 +1,28 @@
 import Quill from 'quill';
 
+// specify the fonts you want
+let fonts = ['Arial', 'Courier', 'Times New Roman', 'Verdana'];
+// generate code friendly names
+function getFontName(font) {
+	return font.toLowerCase().replace(/\s/g, "-");
+}
+let fontNames = fonts.map(font => getFontName(font));
+// add fonts to style
+let fontStyles = "";
+fonts.forEach(function(font) {
+	let fontName = getFontName(font);
+	fontStyles += ".ql-snow .ql-picker.ql-font .ql-picker-label[data-value=" + fontName + "]::before, .ql-snow .ql-picker.ql-font .ql-picker-item[data-value=" + fontName + "]::before {" +
+				"content: '" + font + "';" +
+				"font-family: '" + font + "', sans-serif;" +
+				"}" +
+				".ql-font-" + fontName + "{" +
+				" font-family: '" + font + "', sans-serif;" +
+				"}";
+});
```

Comment:
This part of code has readability issues and looks bad... you can always use [Template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for such cases.
```suggestion
	fontStyles += `
		.ql-snow .ql-picker.ql-font 
		.ql-picker-label[data-value=${fontName]::before,
		.ql-snow .ql-picker.ql-font 
		.ql-picker-item[data-value=${fontName}]::before {
				content: ${font};
				font-family: ${font}, sans-serif;
		}
		.ql-font-${fontName} {
			font-family: ${font}, sans-serif;
		}
	`;
});
```
Note: Test it locally

Reply:
Notes have been fixed.

Reply:
You forgot ' ' in content: ${font}; in order to work it should be like this content: '${font}'; .